pkg/kvstore/allocator/allocator.go

```diff
@@ -438,12 +440,40 @@ func (k *kvstoreBackend) RunGC(ctx context.Context, rateLimit *rate.Limiter, sta
 
 	staleKeys := map[string]uint64{}
 
+	min := uint64(minID)
+	max := uint64(maxID)
+	reasonOutOfRange := fmt.Sprintf("out of local cluster identity range [%d,%d]", min, max)
+
 	// iterate over /id/
 	for key, v := range allocated {
 		// if k.lockless {
 		// FIXME: Add DeleteOnZeroCount support
 		// }
 
+		// Parse identity ID
+		items := strings.Split(key, "/")
+		if len(items) == 0 {
+			log.WithField(fieldKey, key).WithError(err).Warning("Identity GC: unknown identity key, skip it")
+			continue
+		}
+
+		if identityID, err := strconv.ParseUint(items[len(items)-1], 10, 32); err != nil {
+			log.WithFields(logrus.Fields{
+				fieldKey: key,
```

Comment:
Let's add fieldKey to the logfields package.

Reply:
Sorry, I don't get the point, but isn't `fieldKey` already in logfields [here](https://github.com/cilium/cilium/blob/v1.9.5/pkg/kvstore/allocator/logfields.go#L19)?

Reply:
Ah, I missed that. I assumed `fieldKey` was defined locally in this file.